### DISCUSSION TITLE
Where can I find C/2017 K2 on stellarium?

### DISCUSSION THREAD

**Discussion by aedas** [Stellarium/stellarium#2556] [Q&A] [answered]
First time ever joining a github open source software community. Looking forward to learn more and contribute in the future.

I was trying to find the comet C/2017 K2 using the search window, but found no results. I can find other comets using the search though. I found that others can find K2 in the app (see closed issue #2498), so I might be missing something.

Version: 0.22.2
OS: Windows 10

**Comment by gzotti** [comment, marked as answer]
https://github.com/Stellarium/stellarium/wiki/Common-Problems-for-the-current-version#Comet_missing
